utils/libnetwork/Makefile

```diff
@@ -30,7 +31,7 @@ define Package/libnetwork
   CATEGORY:=Utilities
   TITLE:=networking for containers
   URL:=https://github.com/docker/libnetwork
-  DEPENDS:=$(GO_ARCH_DEPENDS)
+  DEPENDS:=$(GO_ARCH_DEPENDS) @TARGET_x86_64
```

Comment:
Actually, a better dependency would be @!(mips||mipsel||mips64||mips64el) since it builds fine elsewhere.

Reply:
I don't have strong feelings either way on this, since docker-ce has [`@TARGET_x86_64`](https://github.com/openwrt/packages/blob/master/utils/docker-ce/Makefile#L47) and I don't know how useful this package is without docker-ce.

Reply:
Changed as suggested by @neheb